kotlin/lib/src/main/kotlin/Application.kt

```diff
@@ -0,0 +1,51 @@
+package com.svix.kotlin
+
+import com.svix.kotlin.exceptions.ApiException
+import com.svix.kotlin.internal.apis.ApplicationApi
+import com.svix.kotlin.models.ApplicationIn
+import com.svix.kotlin.models.ApplicationOut
+import com.svix.kotlin.models.ListResponseApplicationOut
+
+class Application(debugUrl: String = SvixOptions.DEFAULT_URL) {
```

Comment:
This is an internal helper that we use, maybe call it (and others) as such? Like `ApplicationApiWrapper` so it's kind of not really accessible just by the means of having a shit name. :)

Reply:
I'm not sure how each client will use it, but if someone wants to do
val application : ** = svix.application 
they will need to have access to it

Reply:
Yeah, they shouldn't. Only access it through svix.application. Func

Reply:
The class name is accessible as it's returned from Svix.kt, I think it's fine to keep this name, though I changed the constructor to be internal

Reply:
It's confusing with the model itself (as in, you'd think it may be the Application model).

Reply:
Not saying it's a good thing, but this is what we have in Java

I can change their package to model if you think it's better

in the model we currently only have ApplicationApi

Reply:
Cool!